pkg/virt-operator/resource/generate/components/crds.go

```diff
@@ -766,6 +766,17 @@ func NewPrometheusRuleSpec(ns string) *promv1.PrometheusRuleSpec {
 							"summary": "More than 80% of the rest calls failed in virt-handler for the last 5 minutes",
 						},
 					},
+					{
+						Alert: "OrphanedVirtualMachineImages",
+						Expr:  intstr.FromString("(count by(node)(node_namespace_pod:kube_pod_info:{pod=~'virt-launcher-.*'} ) * on (node) group_left(pod) (1*(kube_pod_container_status_ready{pod=~'virt-handler-.*'} + on (pod) group_left(node) (0 * node_namespace_pod:kube_pod_info:{pod=~'virt-handler-.*'} ))) or on (node) (0 * node_namespace_pod:kube_pod_info:{pod=~'virt-launcher-.*'} ) ) == 0"),
+						For:   "60m",
```

Comment:
I don't understand the code, but I see 60m mentioned in two different places.

For testing, I would like to be able to make this alert fire much quicker. Can you explain in the PR text what I should be doing?

Reply:
Our testing at the moment mocks this situation and verifies the rule fires given a specific scenario. It's not a real world functional test where we wait 'x' amount of time for the alert to fire, but a simulation.

For now, that simulation is all we have to work with in CI, and the 60m doesn't impact the time involved with the simulation. I think the important thing is that the simulation verifies the alert formula.